internal/runtime/storeforward.go

```diff
@@ -145,7 +185,7 @@ func (sf *storeForwardInfo) retryStoredData(serviceKey string) {
 		itemsToRemove, itemsToUpdate := sf.processRetryItems(items)
 
 		lc.Debugf(" %d stored data items will be removed post retry", len(itemsToRemove))
-		lc.Debugf(" %d stored data items will be update post retry", len(itemsToUpdate))
+		lc.Debugf(" %d stored data items will be updated post retry", len(itemsToUpdate))
```

Comment:
should these be sf.lc?
```suggestion
		sf.lc.Debugf(" %d stored data items will be updated post retry", len(itemsToUpdate))
```

Reply:
fixed